Github-Org-Management-Policy.md

```diff
@@ -0,0 +1,50 @@
+# Node.js GitHub Organization Management Policy
+
+The Node.js Foundation Github Organization (https://github.com/nodejs) is
+provided as a development resource by the Node.js Foundation under the
+operational direction of the Node.js Technical Steering Committee (TSC).
+
+## Organization Roles
+
+Individual users within a GitHub organization may be granted one of three
+distinct roles: Member, Owner, or Billing manager. The permissions granted
+to each role are documented in the GitHub documentated [Permission levels for
+an organization](https://help.github.com/articles/permission-levels-for-an-organization/).
+
+### Billing manager
+
+All Billing managers for the Node.js Foundation GitHub Organization shall
+be determined by the Node.js Foundation Board of Directors with notification
+given to the TSC.
+
+### Owners
+
+All voting members of the Node.js TSC shall have Owner permissions within the
+Node.js Foundation GitHub Organization. Additional users may be granted Owner
+permissions at the discretion of the TSC through normal TSC motion and simple
+majority vote.
+
+### Members
+
+GitHub users may be granted Member status in the Node.js Foundation GitHub
+Organization by any Owner.
+
+## Repositories
+
+Any organization member may create new repositories within the Node.js
+Foundation GitHub Organization. Any such repository is considered to be a
+project under the ownership of the Node.js Foundation, and thereby subject
+to the Intellectual Property and Governance policies of the Foundation under
+the operational direction of the TSC.
```

Comment:
I see no conflict here with regards to those. They are essentially just spaces carved out to help with general operation of Foundation initiatives. The TSC essentially delegates responsibility for those repositories to the Foundation. If the Foundation needs operational oversight of those (for instance, if the Foundation needs to designate someone with owner permissions in the Org specifically for those repositories), then a simple request to the TSC is made to add that owner.